.github/CODEOWNERS

```diff
@@ -17,9 +17,11 @@
 # Sharmila Ramamoorthy (sharmilarama)
 # Evgeny Uglov (EvgenyUglov)
 # Bowen Jiang (bjiang27)
-# Isaias Martinez (isaiasA1)
 # Arnchie qu (arnchiequ-dell)
```

Comment:
Just to confirm we are still keeping Arnchie?

Reply:
fixed